client/src/components/PageLayout/index.tsx

```diff
@@ -5,13 +5,31 @@ import {
   AlertDescription,
   AlertTitle,
 } from '@chakra-ui/react';
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
There's a way to do what we want without `useEffect`, using `skip` (I'll demonstrate below)